### PR TITLE
np.int, np.bool attributes deprecated for numpy>1.20

### PR DESCRIPTION
**Pull request by dturpin-svom**
minor bug correction due to the upgrade of numpy>1.20 that is no longer compatible with int, float, str, bool attributes. Adding a function in utils to convert float arrays into int arrays

### CODE REVIEW

File: stdpipe/psf.py [hunk 1]
```diff
@@ -352,8 +352,8 @@ def place_psf_stamp(image, psf, x0, y0, flux=1, gain=None):
 
     # Corresponding image pixels
     y1,x1 = np.mgrid[0:stamp.shape[0], 0:stamp.shape[1]]
-    x1 += np.int(np.round(x0) - np.floor(stamp.shape[1]/2))
-    y1 += np.int(np.round(y0) - np.floor(stamp.shape[0]/2))
+    x1 += utils.float2int_array((np.round(x0) - np.floor(stamp.shape[1]/2)))
```
Comment (karpov-sv) [inline stdpipe/psf.py:355]: x0,y0 here are scalars, so probably just `int(np.round(x0) - np.floor(stamp.shape[1]/2))` would go?..

Comment (dturpin-svom) [reply stdpipe/psf.py:355]: Yes, if x0 and y0 are scalars it will work with just int()

Comment (karpov-sv) [reply stdpipe/psf.py:355]: Could you please commit this variant (with just int, and without extra util function) - as I probably cannot commit to your branch. After that I will merge the PR

Comment (dturpin-svom) [reply stdpipe/psf.py:355]: I'll do it asap

**Pull request merged**